.goreleaser.yaml

```diff
@@ -20,6 +20,21 @@ builds:
     ignore:
       - goos: linux
         goarch: '386'
+brews:
+  - name: idpbuilder
+    homepage: "https://cnoe.io"
+    repository:
+      owner: cnoe-io
+      name: homebrew-tap
+      token: "{{ .Env.HOMEBREW_TOKEN }}"
+    commit_author:
```

Comment:
I know this is an automated build and we'd want to call that out, but does it make sense to use something more related to CNOE here? Maybe: 
```
name: "CNOEAutomation"
email: "noreply@cnoe.io"
```

Reply:
Yea I'd want this to be non-generic.